### PR TITLE
release: 3.3.0-cr2

### PR DESCRIPTION
**Pull request by davidor**

### CODE REVIEW

**Comment by davidor** [comment]
Travis fails because the changelog links to the v3.3.0-cr2 tag which will be created after merging.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [3.3.0-cr2] - 2018-09-25
```
Comment (mikz) [inline CHANGELOG.md]: Isn't it rc2?

Comment (davidor) [reply CHANGELOG.md]: :man_facepalming: 
Copy-pasted from rc1 which is wrong.

Comment (davidor) [reply CHANGELOG.md]: let's keep 'rc' for 3.3.0 at least so we don't have to change the existing rc1 tag.

**Pull request merged**